### PR TITLE
Update `setup.py`

### PR DESCRIPTION
**Pull request by curita**
Made some changes in `setup.py` to add some extra details to [Spidermon's PyPI page](https://pypi.org/project/spidermon/).

1. Used the existing README as the `Project description` of the library
2. Added "Documentation", "Source", "Tracker" and "Release notes" as additional `Project links`.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,10 +4,16 @@
...
+        "Documentation": "https://spidermon.readthedocs.io/",
+        "Source": "https://github.com/scrapinghub/spidermon",
+        "Tracker": "https://github.com/scrapinghub/spidermon/issues",
+        "Release notes": "https://spidermon.readthedocs.io/en/latest/changelog.html",
+    },
     author="Zyte",
     author_email="opensource@zyte.com",
     description=("Spidermon is a framework to build monitors for Scrapy spiders."),
-    long_description=("Spidermon is a framework to build monitors for Scrapy spiders."),
+    long_description=open("README.rst", encoding="utf-8").read(),
```
Comment (Gallaecio) [inline setup.py:16]: We are missing it in Scrapy, but you should also set `long_description_content_type="text/x-rst"`

**Review by Gallaecio — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (wRAR) [reply setup.py:16]: That's why I just filed #459

Comment (curita) [reply setup.py:16]: I saw it in other projects but wasn't sure if it was necessary! I just added it to the PR.

**Comment by curita** [comment]
Thank you!! ☺️

**Pull request merged**